src/app/data.service.ts

```diff
@@ -58,4 +58,12 @@ export class DataService {
   public getBand(bandId) : Observable<Band>{
     return this.http.get<Band>('/api/bands/' + bandId);
   }
+
+  public getDeletableJobFamilies() : Observable<JobFamily[]> {
+    return this.http.get<JobFamily[]>('/api/deletableJobFamilies');
```

Comment:
To follow up from my previous comment about RESTful practices, an improvement on this URL would be `api/job-families?deletable=True`

Reply:
But again don't worry about changing this at such a late stage.